openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -537,7 +537,7 @@ Location: https://Wallet.example.org/cb?
 
 ## Authorization Error Response
 
-Authorization Error Response MUST be made as defined in [@!RFC6749].
+Authorization Error Responses MUST be made as defined in [@!RFC6749].
```

Comment:
why plural here?

Reply:
This can be phrased correctly either in the singular or the plural form.  If you prefer singular, make a suggested change to the following and I'll accept.
> **The** Authorization Error Response MUST be made as defined in [@!RFC6749].

Reply:
```suggestion
The Authorization Error Response MUST be made as defined in [@!RFC6749].
```